UI/src/assetTypeConfiguration.js

```diff
@@ -1393,10 +1411,32 @@
             selectedAsset.setPropertyByPublicId('opposite_side_sign', '0');
           }
 
-          var functionFn = _.find(validations, function(validation){ return _.includes(validation.types, parseInt(Property.getPropertyValue('Tyyppi', selectedAsset.get())));});
+          //Validation conditions for signs
+          var signsValidations = _.find(validations, function(validation){ return _.includes(validation.types, parseInt(Property.getPropertyValue('Tyyppi', selectedAsset.get())));});
+          var isValidSign = signsValidations ?  signsValidations.validate(Property.getPropertyValue('Arvo', selectedAsset.get())) : true;
+
+          //Validation conditions for additional signs
+          var isValidAdditionalSign = false;
+          var additionalPanelsProps = Property.getPropertyByPublicId(selectedAsset.get().propertyData, 'additional_panel');
+
+          if (_.isEmpty(additionalPanelsProps.values)) {
+            isValidAdditionalSign = true;
+          } else {
+            isValidAdditionalSign = _.every(additionalPanelsProps.values, function (additionalPanelProp) {
+              var additionalSignsValidations = _.find(validations, function (validation) {
+                return _.includes(validation.types, additionalPanelProp.panelType);
+              });
+
+              var additionalPanelValue = additionalPanelProp.panelValue;
+              var isValidPanelValue = additionalSignsValidations ? additionalSignsValidations.validate(additionalPanelValue, additionalSignsValidations.regexToUse) : true;
+
+              return isValidPanelValue;
+            });
+          }
```

Comment:
I don't think you need to do a if else here. Just start as the validation to true and then if you have additional panels do the the validation to change that boolean 

```javascript
var isValidAdditionalSign = true;
...
if (!_.isEmpty(additionalPanelsProps.values)) {
...
}
```

Reply:
Done